src/components/CardProduct/CardProduct.js

```diff
@@ -0,0 +1,20 @@
+import './CardProduct.css';
+
+const CardProduct = (props) => {
+  return (
+    <div className={'product-card-container'}>
+      <div className={'product-card'}>
+        <div className={'image-container'}>
+          <img src={props.image} alt={props.alt}/>
```

Comment:
Its a good practice to destructure the `props` object instead of accessing the properties with `.`